src/pages/yonghyun/Main/Main.js

```diff
@@ -1,10 +1,161 @@
+                    <div className="friends_name">
+                      <div className="friends_id">1st_id</div>
+                      <div className="friendsdesc">회원님을 위한 추천</div>
+                    </div>
+                    <button className="follow">팔로우</button>
+                  </div>
+                  <div className="people">
+                    <img
+                      className="friends_pic"
+                      alt="recommend2"
+                      src="/images/yonghyun/profile.jpeg"
+                      width="32px"
+                      height="32px"
+                    />
+                    <div className="friends_name">
+                      <div className="friends_id">dydgus0421</div>
+                      <div className="friendsdesc">Instagram 신규가입</div>
+                    </div>
+                    <button className="follow">팔로우</button>
+                  </div>
+                  <div className="people">
+                    <img
+                      className="friends_pic"
+                      alt="recommend3"
+                      src="/images/yonghyun/profile.jpeg"
+                      width="32px"
+                      height="32px"
+                    />
+                    <div className="friends_name">
+                      <div className="friends_id">second_id</div>
+                      <div className="friendsdesc">회원님을 위한 추천</div>
+                    </div>
+                    <button className="follow">팔로우</button>
+                  </div>
+                  <div className="people">
+                    <img
+                      className="friends_pic"
+                      alt="recommend4"
+                      src="/images/yonghyun/profile.jpeg"
+                      width="32px"
+                      height="32px"
+                    />
+                    <div className="friends_name">
+                      <div className="friends_id">asfsdk</div>
+                      <div className="friendsdesc">회원님을 위한 추천</div>
+                    </div>
+                    <button className="follow">팔로우</button>
+                  </div>
+                  <div className="people">
+                    <img
+                      className="friends_pic"
+                      alt="recommend5"
+                      src="/images/yonghyun/profile.jpeg"
+                      width="32px"
+                      height="32px"
+                    />
+                    <div className="friends_name">
+                      <div className="friends_id">kyh0421</div>
+                      <div className="friendsdesc">회원님을 위한 추천</div>
+                    </div>
+                    <button className="follow">팔로우</button>
+                  </div>
+                </div>
+              </div>
+              <div className="desc">
+                <nav id="etc">
+                  <p>소개-도움말-홍보 센터-API-채용-정보-개인정보처리방침</p>
+                  <p>약관-위치-인기 계정-해시태그-언어</p>
```

Comment:
이 부분도 링크 구현을 했으면 좋겠습니다.

Reply:
Link를 사용해서 수정하였습니다~!

```suggestion
                  <p><Link className="link" to="#n">
                  소개
                </Link></p>
```